### PR TITLE
Bugfix - IE11 - cannot delete draft

### PR DESCRIPTION
**Pull request by leksmall**
*Issue #:* /bcgov/entity#1853, /bcgov/entity#2037, /bcgov/entity#1805

*Description of changes:*
Added min-width to the list item so that the delete draft button is visible

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
I noticed this bug occurs in other places. Can you fix those too please (and update the ticket accordingly)?
1. Update business profile drop-down
2. Directors change drop-down
3. Cancel payment (I cannot see this yet so I am assuming)

**Comment by leksmall** [comment]
> I noticed this bug occurs in other places. Can you fix those too please (and update the ticket accordingly)?
> 
> 1. Update business profile drop-down
> 2. Directors change drop-down
> 3. Cancel payment (I cannot see this yet so I am assuming)

Cancel Payment and Delete Draft uses the same style. So it works. The other component fixes are added to this PR now. Will link the tickets for them in this PR

File: coops-ui/src/components/AnnualReport/Directors.vue [hunk 1]
```diff
@@ -1548,6 +1548,10 @@ ul {
 .actions__more-actions {
   padding: 0;
 
+  .v-list-item {
+    min-width: 160px;
```
Comment (severinbeauvais) [inline coops-ui/src/components/AnnualReport/Directors.vue]: Looks like the menuitem could use some vertical centering also.
drop down

File: coops-ui/src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -707,6 +707,10 @@ export default {
 .actions__more-actions {
   padding: 0;
 
+  .v-list-item {
+    min-width: 140px;
```
Comment (severinbeauvais) [inline coops-ui/src/components/Dashboard/TodoList.vue]: Too wide. Try 112px.

File: coops-ui/src/components/AnnualReport/Directors.vue (hunk 1 above)
Comment (severinbeauvais) [reply coops-ui/src/components/AnnualReport/Directors.vue]: Please also check v centering on others.

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 2 above)
Comment (leksmall) [reply coops-ui/src/components/Dashboard/TodoList.vue]: This is to take care of Cancel Payment

**Pull request merged**